e2e/06-other-tests.e2e.spec.ts

```diff
@@ -45,7 +44,7 @@ describe("Change Language Flow", () => {
 
   it("navigates back to move home screen", async () => {
     await clickBackButton()
-    await waitTillSettingDisplayed(enLL.common.account())
+    await waitTillTextDisplayed(enLL.common.preferences() + "-group")
```

Comment:
I think we should avoid manipulation string that represent object and ensure there is a proper to target them (without having to add `-group`).